tests/microbenchmarks/skip_softmax/generate_report.py

```
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+"""Render the kernel microbench CSVs into a single markdown report.
+
+Reads ``pass1_sparsity.csv`` (skipped/total per shape × threshold) and
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)